project/project.go

```diff
@@ -94,12 +94,13 @@ func (c projectResolver) Parse(workingDir string, ignoreMissingHalfpipeFile bool
 		return p, err
 	}
 
-	halfpipeFilePath, e := filechecker.GetHalfpipeFileName(c.Fs, workingDir)
+	halfpipeFilePath, e := filechecker.GetHalfpipeFileName(c.Fs, workingDir, halfpipeFilenameOptions)
 	if e != nil {
-		if !(e == errors2.NewMissingHalfpipeFileError() && ignoreMissingHalfpipeFile) {
-			err = e
-			return p, err
-		}
+		// TODO fix this
+		//if !(e == errors2.NewMissingHalfpipeFileError() && ignoreMissingHalfpipeFile) {
```

Comment:
I don't know how to do this check here now that the struct `MissingHalfpipeFileError` has a "field".

Any idea?

Reply:
Resolved by using a switch-case (thanks @simonjohansson)